app/helpers/redirect_helper.rb

```diff
@@ -1,5 +1,5 @@
 module RedirectHelper
```

Comment:
Sorry, misread you comment at first. Your right, and I have now reverted the changes to this file in 99e250f.